foundationdb/datadog_checks/foundationdb/check.py

```diff
@@ -13,9 +13,12 @@
 class FoundationdbCheck(AgentCheck):
     def __init__(self, name, init_config, instances):
         super(FoundationdbCheck, self).__init__(name, init_config, instances)
+        self.fdb_status = init_config.get('fdbstatus_path')
 
     def fdb_status_data(self):
-        return get_subprocess_output(['fdbcli', '--exec', 'status json'], self.log)
+        fdb_args = self.fdb_status.split(' ')
```

Comment:
Hm, this will be vulnerable to messing up paths that are quoted because they contain spaces, I think?

Reply:
We do this split only for this string:
>E2E_INIT_CONFIG = {'fdbstatus_path': 'docker exec fdb-0 fdbcli --exec'}

And there are no paths included, so it should be pretty safe. Of course, we can change it into a list from the start to avoid this conversion, I guess, should we? I relied on the glusterfs implementation.

Reply:
I'd do it as a list from the start, but let's defer that into the overall config changes that you're going to be doing next, perhaps?

The problem isn't our E2E setup where we control it, it's when folks provide their own command line. Granted spaces in such paths aren't *that* likely to happen, but I'd rather bulletproof it.

Reply:
Makes sense, I'll change it here, not a big deal.

Reply:
Done.